compiler/stable_mir/src/ty.rs

```diff
@@ -86,6 +86,25 @@ impl Debug for Span {
     }
 }
 
+impl Span {
+    /// Return filename for diagnostic purposes
+    pub fn get_filename(&self) -> Filename {
+        with(|c| c.get_filename(self))
+    }
+
+    /// Return lines that coresspond to this `Span`
+    pub fn get_lines(&self) -> Vec<LineInfo> {
+        with(|c| c.get_lines(&self))
+    }
+}
+
+#[derive(Clone, Copy, Debug)]
+pub struct LineInfo {
+    pub line_index: usize,
```

Comment:
document whether the first line is `0` or whether it is `1`. Similarly for `col`

Reply:
made them start from 1 now